.changeset/thirty-scissors-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
How does this file get used? I'm not familiar with the @changesets package.

Reply:
I'll show you in our 1:1 today